exercises/F-predicates/exercise.js

```diff
@@ -6,10 +6,16 @@
 */
 
 // Finish the predicate function to test if the passed number is negative (less than zero)
-function isNegative(number) {}
+function isNegative(number) {
+  if(number>0)return false;
```

Comment:
According to this logic `0` is `Negative` which is not quite right. Please have a second look at your comparison sign